GDJS/Runtime/pixi-renderers/spriteruntimeobject-pixi-renderer.js

```diff
@@ -100,6 +100,11 @@ gdjs.SpriteRuntimeObjectPixiRenderer.prototype.setColor = function(rgbColor) {
    this._sprite.tint = "0x" + gdjs.rgbToHex(parseInt(colors[0], 10), parseInt(colors[1], 10), parseInt(colors[2], 10));
 };
 
+gdjs.SpriteRuntimeObjectPixiRenderer.prototype.getColor = function() {
```

Comment:
We would need to implement this for Cocos2D-JS, otherwise we'll get crash of the game when using getColor.